src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotSuitableSDOCallbackHandler.java

```diff
@@ -0,0 +1,101 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.model.BusinessProcess;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.sdo.ReasonNotSuitableSDO;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static java.lang.String.format;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_START;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.MID;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.SUBMITTED;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.NotSuitable_SDO;
+
+@Service
+@RequiredArgsConstructor
+public class NotSuitableSDOCallbackHandler extends CallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = Collections.singletonList(NotSuitable_SDO);
+
+    public static final String NotSuitableSDO_CONFIRMATION_BODY = "<br />If a Judge has submitted this information, "
+        + "a notification will be sent to the listing officer to look at this case offline."
+        + "%n%nIf a legal adviser has submitted this information a notification will be sent to a judge for review.";
+
+    private final ObjectMapper objectMapper;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return new ImmutableMap.Builder<String, Callback>()
+            .put(callbackKey(ABOUT_TO_START), this::prePopulateReasonNotSuitableSDODetailsPages)
+            .put(callbackKey(ABOUT_TO_SUBMIT), this::submitNotSuitableSDO)
+            .put(callbackKey(SUBMITTED), this::buildConfirmation)
+            .build();
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse prePopulateReasonNotSuitableSDODetailsPages(CallbackParams callbackParams) {
```

Comment:
If we don't have a requirement to pre-populate the input field then this entire function is not needed since it isn't really doing anything.